CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   default values pre-populated.  
 
 ### Changed
+- Generated `ToWire` methods no longer mutate the receiver with default values, 
+  leaving nil fields as nil on the object though serializing the defaults.   
```

Comment:
I consider this a bugfix, honestly. How about,

> Serializing generated Thrift objects no longer mutates them with default values for unspecified fields.

Reply:
👌 👍